### DISCUSSION TITLE
Rolladenbezeichnung an mqtt senden?

### DISCUSSION THREAD

**Discussion by Kenaschon** [dewenni/ESP32-Jarolift-Controller#9] [General]
Hallo dewenni,

gibt es eine Möglichkeit, die Kanalbezeichnungen die in den Einstellungen unter "Rolladen" eingegeben werden per mqtt auch mit zu versenden?

Grüße
Manfred

**Comment by dewenni** [comment]
Wie genau meinst du das?
Beim Status oder auch als Kommando?

**Comment by Kenaschon** [comment]
Hier mal das als Bilder

Diese Bezeichnungen
![2](https://github.com/user-attachments/assets/ab104bc9-7103-4b60-9532-0371ff27b62f)

nach hier
![1](https://github.com/user-attachments/assets/a749cd32-762f-495b-852f-015a776a9981)

**Reply by dewenni** [reply]
Möglich wäre das zwar schon, aber das macht es insgesamt schwieriger in der Handhabung.
Auch in Bezug auf Home Assistant. Ich denke ich werde das vorerst nicht ändern.
